ROADMAP.rst

```diff
@@ -6,6 +6,5 @@
 - Add `matchvar` method to lint vars in vars.yml #354
 - Add support for include_tasks which currently are skipped #362
 - Consider adopting/absorbing ansible-lint-junit #396
-- Use an API token for uploading to PyPI from Travis CI instead of account creds #553
```

Comment:
Just change Travis CI  to GHA. Will also need to close that issue once complete.